CHANGELOG

```diff
@@ -2,6 +2,40 @@
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.4.0] - 2023-03-27
+### Added
+- Added config to disable active RED encoding. Use NACK instead #1476 #1477
+- Added option to skip TCP fallback if TCP RTT is high #1484
+- psrpc based signaling between signal and RTC #1485
+- Connection quality algorithm revamp #1490 #1491 #1493 #1496 #1497 #1500 #1505 #1507 #1509 #1516 #1520 #1521 #1527 #1528 #1536
+- Support for topics in data channel messages #1489
+- Added active filter to ListEgress #1517
+- Handling for React Native and Rust SDK ClientInfo #1544
+
+### Fixed
+- Fixed unsubscribed speakers stuck as speaking to clients #1475
+- Do not include packet in RED if timestamp is too far back #1478
+- Prevent PLI layer lock getting stuck #1481
+- Fix a case of changing video quality not succeeding #1483
+- Resync on pub muted for audio to avoid jump in sequence numbers on unmute #1487
+- Fixed a case of data race #1492
+- Inform reconnecting participant about recently disconnected users #1495
```

Comment:
this is probably not a fix, maybe belongs in added.

Reply:
it's probably a terminology difference.. You are right it's a new functionality added to the server. Though I'm thinking fixed in terms of fixing a bug that clients are experiencing. (not seeing disconnected participants disappear).